### ISSUE TITLE
Question about documentation.

### ISSUE THREAD

**Issue by kosuke1701** [kajyuuen/pytorch-partial-crf#11]
Hello. Thank you for sharing useful codes!
I have a question about the following description in README.md.

```python
# Computing log likelihood
model(emissions, tags)
```

Is it log-likelihood of the given tag sequence? Looking at the implementation, I think the output of this function is actually *negative* log-likelihood.

**Comment by kajyuuen** [comment]
As you say, the loss function is negative log-likelihood.
I fixed the documentation.
Thank you for the suggestion.
